pykokkos/core/runtime.py

```diff
@@ -124,6 +125,13 @@ def execute(
         else:
             args["pk_kernel_name"] = name
 
+        red_view_type = kokkos.double
+        for key, val in args.items():
+            if 'reduction_result_' in key:
+                # TODO: other types
+                if  "int64" in str(val):
```

Comment:
This feels pretty fragile--there must be something better than looking at the `repr` of the Kokkos objects for type analysis here.

Reply:
Would `isinstance()` work here?